website/docs/docs/deploy/model-notifications.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Model notifications"
+description: "Receive notifications about failures with your models. "
+---
+
+# Model notifications <Lifecycle status="beta" />
```

Comment:
Check with @rviswanathan-dbt on whether versionless is required here.

Reply:
Yes only versionless for now, this change has not been backported to any other version yet